### ISSUE TITLE
Crash while dispatching a job

### ISSUE THREAD

**Issue by Cellard** [getsentry/sentry-laravel#483]
Using `laravel/framework ^8.0`

1. Install `sentry/sentry-laravel ^2.5`
2. Create empty job.
3. Create unit test. Dispatch the job.

```
TypeError : Argument 2 passed to Sentry\Laravel\Tracing\EventHandler::Sentry\Laravel\Tracing\{closure}() must be of the type string, null given
 .../vendor/sentry/sentry-laravel/src/Sentry/Laravel/Tracing/EventHandler.php:151
 .../vendor/laravel/framework/src/Illuminate/Queue/Queue.php:280
 .../vendor/laravel/framework/src/Illuminate/Queue/Queue.php:151
 .../vendor/laravel/framework/src/Illuminate/Queue/Queue.php:127
 .../vendor/laravel/framework/src/Illuminate/Queue/Queue.php:105
 .../vendor/laravel/framework/src/Illuminate/Queue/SyncQueue.php:38
 .../vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:251
 .../vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:227
 .../vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:77
 .../vendor/laravel/framework/src/Illuminate/Foundation/Bus/PendingDispatch.php:202
 .../vendor/laravel/framework/src/Illuminate/Foundation/Bus/Dispatchable.php:17
 .../tests/Unit/ExampleTest.php:20
```

**Comment by stayallive** [comment]
Hi @Cellard, would you be so kind to confirm the fix in master is working?

For in your `composer.json` to pull in master:

```json
	"sentry/sentry-laravel": "dev-master#cae4521c2163334ec06bd2b9c970c32fbd369da8"
```

**Comment by Cellard** [comment]
It works

**Comment by stayallive** [comment]
Thanks, fixed release (2.5.1) should be out soon!

As a side note, you can disable Sentry in unit tests by setting `SENTRY_LARAVEL_DSN` to `null`, this might speed up your tests and prevent issues like this (although it still should not have failed of course).
